src/components/atoms/Flex/README.md

```diff
@@ -0,0 +1,9 @@
+# Flex
+[![Generic badge](https://img.shields.io/badge/GROUP-global-<COLOR>.svg)]()
+[![Generic badge](https://img.shields.io/badge/SIZE-atom-blue.svg)]()
+
+Flex. This component is a global component.
+
+## Usage
+
+## Properties
```

Comment:
This README can be deleted as it's unused

Reply:
I will update the file